src/GiftedChat.tsx

```diff
@@ -594,6 +596,11 @@ class GiftedChat<TMessage extends IMessage = IMessage> extends React.Component<
 
   onKeyboardWillShow = (e: any) => {
     if (this.props.isKeyboardInternallyHandled) {
+      if (new Date().getTime() - this._lastUpdate < 100) {
```

Comment:
Hi,
Not tested it yet but seems a good idea.
A small suggestion maybe using Date.now():
```suggestion
      if (Date.now() - this._lastUpdate < 100) {
```